cmd/mimir/config-descriptor.json

```diff
@@ -3096,6 +3096,17 @@
           "fieldType": "boolean",
           "fieldCategory": "experimental"
         },
+        {
+          "kind": "field",
+          "name": "ignore_native_histograms_on_read",
+          "required": false,
+          "desc": "True to stop processing native histograms as soon as possible.",
```

Comment:
Is this an information statement or an action? If it is an action, I recommend, "To stop processing native histograms as soon as possible, set to `true`."

If feedback is irrelevant, kindly ignore.

Reply:
Thanks for the suggestion--I'll take it!